test/benchmark/shell_scripts/iGenVar_statistics_via_truvari.sh

```diff
@@ -0,0 +1,57 @@
+# -------- -------- pre installation steps -------- -------- #
+# for truvari you need python3,
+# we recommend to use miniconda:
+# ------------------------------
+# wget https://repo.anaconda.com/miniconda/Miniconda2-latest-Linux-x86_64.sh
+# chmod +x Miniconda2-latest-Linux-x86_64.sh
+# sh Miniconda2-latest-Linux-x86_64.sh
+
+# conda create -n benchmark -c defaults -c bioconda python=3 pip tabix samtools
+# pip install truvari
```

Comment:
You could also provide an `environment.yml` (name can be whatever) that you then can use:

```yml
name: iGenVar_benchmark
channels:
  - defaults
  - conda-forge
  - bioconda
dependencies:
  - python=3
  - samtools
  - tabix
  - pip
  - pip:
    - truvari
```

Env name `iGenVar_benchmark` would be better as it is not as generic.

Then you can just use `conda env create -f environment.yml`

I guess you could then always do:
```bash
conda env create -f environment.yml
conda activate benchmark
```

then you don't have to worry as much about setup